HelpGuide.md

```diff
@@ -0,0 +1,81 @@
+# `Help Guide` Cheat Sheet
+
+## Introduction
+
+Welcome to the Cheat Sheet Help Guide. This guide is designed to help you navigate the website and make the most of the features and resources offered. The cheat sheets are packed with valuable information, and we want to ensure you have all the tools you need to use them effectively.
+
+### What's Inside This Guide
+1. Basic Navigation
+2. Features
+3. FAQ
+4. Troubleshooting Guide
+5. Feedback and Support
```

Comment:
It'd be nice for these to be anchor links to the referenced sections

Reply:
Added anchor links to the sections